Sources/Model/Receipt/HyperwalletReceipt.swift

```diff
@@ -168,9 +168,10 @@ public struct HyperwalletReceipt: Decodable, Equatable {
         case prepaidCardPinChangeFee = "PREPAID_CARD_PIN_CHANGE_FEE"
         case prepaidCardRefund = "PREPAID_CARD_REFUND"
         case prepaidCardReplacementFee = "PREPAID_CARD_REPLACEMENT_FEE"
-        case pPrepaidCardSale = "PREPAID_CARD_SALE"
+        case prepaidCardSale = "PREPAID_CARD_SALE"
```

Comment:
move this to the last before PayPal section. This section is for Prepaid Card

Reply:
Moved